spec/e2e/pbivizNewSpec.js

```diff
@@ -110,7 +110,8 @@ describe("E2E - pbiviz new", () => {
                 visualPath = path.join(tempPath, visualName);
 
             FileSystem.runPbiviz('new', visualName, `--template ${template}`);
```

Comment:
I'm not sure about it. I think we should left it for user - to run npm install command